core/src/test/java/org/opensearch/sql/analysis/AnalyzerTest.java

```diff
@@ -553,7 +554,7 @@ public void project_nested_field_arg() {
     List<NamedExpression> projectList =
         List.of(
             new NamedExpression(
-                "message.info",
+                "nested(message.info)",
```

Comment:
do we want to test both of these cases?

Reply:
No need, this change should've been made with the original `SELECT` PR. This is due to the difference that the V1 engine uses only the field name as the column identifier while the V2 engine uses the function name with field value.

V1 Identifier: `message.info`
V2 Identifier: `nested(message.info)`